static/app/views/dashboardsV2/widget/issueWidget/utils.tsx

```diff
@@ -24,3 +25,15 @@ export function generateIssueWidgetFieldOptions(
 
   return fieldOptions;
 }
+
+export function generateIssueWidgetOrderOptions() {
+  const options: SelectValue<string>[] = [];
+  for (const key in IssueSortOptions) {
```

Comment:
Comparing the sort options on the Issues list page with the new field, the field has two new options

**List page:**
![Screen Shot 2021-12-14 at 3 40 13 PM](https://user-images.githubusercontent.com/22846452/146076288-2bc29d87-9661-4c81-a779-7b93b64cc6d9.png)

**Field:**
![Screen Shot 2021-12-14 at 3 40 19 PM](https://user-images.githubusercontent.com/22846452/146076280-57bce851-ec55-494b-85d3-7923b792424f.png)

The last one triggers a 500. Not sure why we have these two options we don't show but for now I think we might want to return the same options as the Issues page

Reply:
Good catch, I've updated the sort by list to only include values from the Issues page dropdown. `Relative Change` is also included, conditional depending on the organizations feature flag.

![image](https://user-images.githubusercontent.com/83961295/147949179-7acf717d-cfc1-4ac8-abeb-624b907424de.png)